src/test-interp.cc

```diff
@@ -505,8 +505,13 @@ TEST_F(InterpTest, Rot13) {
 
     EXPECT_LT(ptr + size, memory->ByteSize());
 
+#if WABT_BIG_ENDIAN
+    std::copy(string_data.rbegin(), string_data.rbegin() + size,
+              memory->UnsafeData() + memory->ByteSize() - ptr - size);
+#else
     std::copy(string_data.begin(), string_data.begin() + size,
               memory->UnsafeData() + ptr);
+#endif
```

Comment:
It it worth extracting a named function here?

Reply:
it's the only test that needs this (*currently) so probably not yet?

Reply:
What I mean is that this logic seems to be duplicated with lines 535 to 542 below.   Would a "CopyUserMemory" helper function make sense?

Reply:
not in this context, these are slightly different operations (copy to memory vs copy from memory) and we can't think of a good way to abstract over both of them.

the APIs between host and wasm are rather lacking in pretty much every standalone wasm engine we've tried. allegedly interface types were supposed to help with this but uh... wabt doesn't really have those yet. (and probably won't for a while...)